python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -430,6 +433,12 @@ def agg(self, func):
 
         Specifying a list of aggregations to perform on each column.
 
+        >>> import cudf
+        >>> a = cudf.DataFrame({
+        ...     'a': [1, 1, 2],
+        ...     'b': [1, 2, 3],
+        ...     'c': [2, 2, 1]
+        ... })
         >>> a.groupby('a').agg(['sum', 'min'])
             b       c
           sum min sum min
```

Comment:
```suggestion
                b       c
          sum min sum min
```

While we are at it, can we properly indent the first lines of all the column names in this file? These are translating to improper indentation on the actual pages too, for example: https://docs.rapids.ai/api/cudf/nightly/api_docs/api/cudf.core.groupby.groupby.groupby.agg#cudf.core.groupby.groupby.GroupBy.agg